packages/confirm-dialog/src/vaadin-confirm-dialog-mixin.js

```diff
@@ -288,51 +291,46 @@ export const ConfirmDialogMixin = (superClass) =>
         },
       });
       this.addController(this._confirmController);
+    }
 
-      this._overlayElement = this.$.overlay;
```

Comment:
Let's restore this for now as it's used by `OverlayClassMixin` to propagate overlay classes (without this property, it won't work). We could pass classes in Lit template instead but that's a separate topic.

Reply:
Sorry, missed this when updating the snapshots. Restored it.